src/main/java/monitoringsystemturbo/Main.java

```diff
@@ -25,62 +26,61 @@
 
     @Override
     public void start(Stage primaryStage) throws Exception {
-        Parent root = FXMLLoader.load(getClass().getResource("/sample.fxml"));
-        primaryStage.setTitle("Hello World");
-        primaryStage.setScene(new Scene(root, 300, 275));
-        primaryStage.show();
+        primaryStage.setTitle("MonitoringSystemTurbo");
+        MainController mainController = new MainController(primaryStage);
+        mainController.showMainWindow();
     }
 
     public static void main(String[] args) {
```

Comment:
All well and good, but maybe you should  create second main insted of commenting everything.